code/game/gamemodes/wizard/wizloadouts.dm

```diff
@@ -99,3 +99,23 @@
 	wizid.SetOwnerInfo(user)
 	wizid.UpdateName()
 	wizid.RebuildHTML()
+
+/datum/spellbook_entry/loadout/oblivion
+	name = "Oblivion Enforcer"
+	desc = "The Oblivion Order is an isolated clique of monks that revere the supermatter. \
+	Oblivion Enforcers are how the Order imposes their will on the universe as a whole. By taking this loadout, \
+	you give up your identity and become a faceless hand of the Order. <br>\
+	You will be completely protected from the effects of the supermatter by the items granted here, so far as to \
+	allow you to pick up and throw supermatter slivers, which your halberd can cut from the engine. <br>\
+	</i>Provides a Supermatter Halberd, Oblivion Enforcer robes, and an air tank, as well as Instant Summons, Lightning Bolt, Ethereal Jaunt, and Summon Supermatter Crystal.<i>"
```

Comment:
```suggestion
	desc = "The Oblivion Order is an isolated clique of monks that revere supermatter crystals. \
	Oblivion Enforcers are how the Order imposes their will on the universe as a whole. By taking this loadout, \
	you give up your identity and become a faceless hand of the Order. <br>\
	You will be completely protected from the effects of supermatter crystals by the items granted here, so far as to \
	allow you to pick up and throw supermatter slivers, which your halberd can cut from the engine. <br>\
	</i>Provides a Supermatter Halberd, Oblivion Enforcer robes, and an air tank, as well as Instant Summons, Lightning Bolt, Ethereal Jaunt, and Summon Supermatter Crystal.<i>"
```
This might sound a bit better? Might be a bit understandable because `the supermatter` is a very broad and vague concept.